src/gui/sdl_mapper.cpp

```diff
@@ -2099,10 +2136,7 @@ static void SetActiveEvent(CEvent * event) {
 		mapper.abindit=event->bindlist.begin();
 		if (mapper.abindit!=event->bindlist.end()) {
 			SetActiveBind(*(mapper.abindit));
-
-			const auto new_bind = dynamic_cast<CKeyBind*>(*mapper.abindit);
-			drop_other_bound_events(new_bind);
-
+			drop_other_bound_events(*mapper.abindit);
```

Comment:
Minor but I really consistency:  Remove the parenthesis in that top line.  Member access happens before the de-reference so it's not needed.  Should just be `SetActiveBind(*mapper.adindit);`

I only just now noticed it because you added that last line that didn't have the extra parenthesis.